CONTRIBUTING.md

```diff
@@ -27,6 +28,59 @@ The following document is a rule set of guidelines for contributing.
+| Bugs in IF core leading to incorrect calculations                                                                      | unusable framework                                                                       | 5        |
+| Bugs in if-plugins leading to incorrect calculations                                                                   | core pathways fail, IF very limited in functionality                                     | 5        |
+| Bugs in if-unofficial-pluginsd leading to incorrect calculations                                                       | Third party plugins harder to use, limits IF to standard lib                             | 3        |
+| Bugs in template                                                                                                       | Harder to build plugins, ecosystem growth is impacted                                    | 2        |
+| Bugs in docs                                                                                                           | product does not match expectation, hard to debug, frustration, loss of adoption         | 2        |
+| Security flaw: privacy related                                                                                         | leak user data, unlikely to achieve adoption in serious orgs                             | 5        |
+| Security flaw: permissions escalation                                                                                  | expose user to malware                                                                   | 5        |
+| Code not addressing user needs                                                                                         | no product market fit, loss of adoption                                                  | 5        |
+| Communication failures within team                                                                                     | Conflicting or duplicating work, frustration, morale damage                              | 4        |
+| Communication failures with community                                                                                  | we lose product market fit, we do not have good community retention, reputational damage | 3        |
+| Communication failures with leadership                                                                                 | product does not meet business goals                                                     | 3        |
+| License compliance failures, including in supply chain (e.g. exposing privileged api responses for free via  a plugin) | 4                                                                                        |
+| Bugs affecting releases                                                                                                | users stuck on old versions                                                              | 2        |
```

Comment:
@jmcook1186 in IF-docs I wrote a question about the severity level of "bugs affecting releases" . In case it is changed, please update it here as well

Reply:
```suggestion
| Bugs affecting releases                                                                                                | users stuck on old versions                                                              | 4        |
```